### PR TITLE
Audit log to an audit destination

### PR DESCRIPTION
**Pull request by dheniges**
Requires #298 

Uses a separate logger instance for insight ops so we can group and track audit logs separately. They have an API we can use to download the logs and save them on S3. Opening a new ticket for that work.

Audit logging defaults to console if no audit log token provided.

### CODE REVIEW

File: api/src/integrators/audit.js [hunk 1]
```diff
@@ -1,6 +1,19 @@
 // Controller audit log integrator
 // receives an object with namespace info
 // currently logging to the console
```
Comment (jjidt) [inline api/src/integrators/audit.js]: I think we can update the comment here now

**Review by jjidt — CHANGES_REQUESTED** [review]
Just updating the comment, everything looks good 👍

File: api/src/integrators/audit.js (hunk 1 above)
Comment (dheniges) [reply api/src/integrators/audit.js]: updated

**Review by jjidt — APPROVED** [review]
👍

**Pull request merged**